### ISSUE TITLE
possibility to proxify a UWP(ms-store) app?

### ISSUE THREAD

**Issue by crazii** [shunf4/proxychains-windows#29]
So far I've tried:
`proxychains_win32_x64 cmd /c start [/wait] netflix:`
`proxychains_win32_x64 explorer netflix:`
this method works for win32 applications such as chrome, msedge

but it is not working for ms-store apps, and I got these message while the app is still running:
> [I] 2021/07/12 23:48:33 All Windows descendant process exited.
> [I] 2021/07/12 23:48:33 Master exiting

It seems the UWP app is hookable ([stackoverflow:inject-a-dll-file-to-uwp](https://stackoverflow.com/questions/52833300/inject-a-dll-file-to-uwp)) but the hooking chain is broken between win32 executable & UWP sandbox container, as the UWP apps have different launch mechanisms.

**Comment by shunf4** [comment]
Yeah it seems possible. But UWP Sandbox limitations will void many of the mechaisms we are using (network, IPC etc), so there's a lot of work to do.

**Comment by crazii** [comment]
> Yeah it seems possible. But UWP Sandbox limitations will void many of the mechaisms we are using (network, IPC etc), so there's a lot of work to do.

Okay. Is there any plan to add this?
I thinks there're not many requirements, so I don't expect much that it will be added.

**Comment by shunf4** [comment]
> Okay. Is there any plan to add this?
No.
